presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java

```diff
@@ -145,13 +155,32 @@ public void testNormalQuery()
                 .mapToLong(e -> e.getStatistics().getCompletedPositions())
                 .sum();
 
-        MaterializedResult result = queryRunner.execute(session, "SELECT count(*) FROM lineitem");
+        MaterializedResult result = runQueryAndWaitForEvents("SELECT count(*) FROM lineitem", expectedEvents, session);
         long expectedCompletedPositions = (long) result.getMaterializedRows().get(0).getField(0);
 
         assertEquals(actualCompletedPositions, expectedCompletedPositions);
         assertEquals(queryCompletedEvent.getStatistics().getTotalRows(), expectedCompletedPositions);
     }
 
+    @Test
```

Comment:
Can we simply add a check for resource group into existing tests? Also, there's no need for event-client-test RG, let's just use the user.